Modules/Module_Processing/xASL_init_LoadDataPar.m

```diff
@@ -134,4 +136,61 @@
     %% Load dataPar
     x = xASL_adm_MergeStructs(dataPar.x, x);
     
-end
\ No newline at end of file
+end
+
+%% -----------------------------------------------------------------------------
+%% -----------------------------------------------------------------------------
+function [x] = xASL_initLoadDataPar_PrepareAtlas4ROI(x)
+%Check for x.S.Atlases & x.S.TissueMasking and provide default values or print instructions
+
+bAtlasTissueMatch = true; % Necessary condition
```

Comment:
rename comment to 

% x.S.Atlases & x.S.TissueMasking should match before we can continue (needed for the population module)

"necessary condition" confused me at least :)

Reply:
It was you who wrote that :) I just copy-pasted it here :)

But sure, changed...